백세희/index.js

```diff
@@ -0,0 +1,16 @@
+const display = document.querySelector(".js-display");
+const result = document.querySelector(".js-result");
+
+function add(char) {
+  display.value += char;
+}
+
+function calculate() {
+  const RESULT = eval(display.value);
+  document.querySelector(".js-result").value = RESULT;
```

Comment:
```suggestion
  result.value = RESULT;
```

이 부분에 미리 선언해 둔 변수 (DOM)을 사용하지 않으신 이유가 무엇인가요??

Reply:
result랑 RESULT 이름이 헷갈려 보여서 그랬습니다. 미리 선언한 변수를 사용하는 것이 맞아요!